public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -101,9 +101,9 @@ <h5 class="col-xs-12 box-title">
     </div>
 
     <div class="col-md-8 col-sm-6 col-xs-12 chart-wrapper flex-xs-middle">
-      <app-chart [concepts]="top10Results" [chartTitle]="'Top 10 by particpant count'" [pointWidth]="15" [backgroundColor]="'#D9E4EA'"> </app-chart>
+      <app-chart [concepts]="top10Results" [chartTitle]="'Top 10 by participant count'" [pointWidth]="15" [backgroundColor]="'#D9E4EA'"> </app-chart>
```

Comment:
Do you have to change this file? I have changed it a bunch and may get merge conflicts. I've fixed these issues in my branch.